include/vcpkg/packagespec.h

```diff
@@ -92,6 +93,12 @@ namespace vcpkg
         std::string m_feature;
     };
 
+    /// In an internal feature set, "default" represents default features and missing "core" has no semantic
+    struct InternalFeatureSet : std::vector<std::string>
```

Comment:
why are we inheriting from a standard type here?

Reply:
I wanted existing algorithms to work and didn't see value in forwarding all the member functions.

This is (almost) purely annotation; in all other ways it should act like a vector.